.github/workflows/ubuntu.yml

```diff
@@ -6,6 +6,70 @@ env:
   CFLAGS: -O2 -Wformat -Wformat-security -Wall -Werror -D_FORTIFY_SOURCE=2 -fstack-protector-strong
 
 jobs:
+  clang12-api-latest:
```

Comment:
@onabiull : do we need to be over paranoid and have both clang-11 and clang-12 at the same time? I initially thought to just switch clang-11 configuration to clang-12...

Reply:
In this case we need remove clang 10 as well...
I thought that we only ADD builds when found it.

Reply:
After offline talk: we'll go with modifying clang-11 configuration

Reply:
DONE